Yoast/yoast-components
# Implement cross with onClick functionality inside input container

## Summary

This PR can be summarized in the following changelog entry:

* Keyword input fields now have a button that allows removal of that keyword section.

## Relevant technical choices:

* Styling the input field and overriding Gutenberg styles was very cumbersome (increasing specificity with `&&&` did not work), so for now we have used `! important`. @moorscode is doing a big styling pass, perhaps this can be fixed once he found a decent solution.

## Test instructions

This PR can be tested by following these steps:

* Verify in the yoast-components examples (`localhost:3333`), under keyword, that the KeywordInput now has a cross. If you click it, the console should log "CLOSED", as that is the onClick functionality we added to the example.
